src/utils/ScorecardFilterUtils.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 Cortex Applications, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CategoryFilter, FilterType, Scorecard } from "../api/types";
+
+export function isScorecardTeamBased(scorecard?: Scorecard): boolean {
+  return (
```

Comment:
Could we break up this logic into multiple `if` conditions to make it easier to read please? 🙏